docs/sdk/src/guides/your_first_runtime.rs

```diff
@@ -1,3 +1,169 @@
 //! # Your first Runtime
 //!
-//! 🚧 <https://github.com/paritytech/polkadot-sdk/pull/3946>
+//! This guide will walk you through the steps to add your pallet to a runtime.
+//!
+//! The good news is, in [`crate::guides::your_first_pallet`], we have already created a _test_
+//! runtime that was used for testing, and a real runtime is not that much different!
+//!
+//! ## Setup
+//!
+//! A runtime shares a few similar setup requirements as with a pallet:
+//!
+//! * importing [`frame`], [`codec`], and [`scale_info`] crates.
+//! * following the [`std`](crate::polkadot_sdk::substrate#wasm-build) pattern.
+//!
+//! But, more specifically, it also contains:
+//!
+//! * a `build.rs` that uses [`substrate_wasm_builder`]. This entails declaring
+//!   `[build-dependencies]` in the Cargo manifest file:
+//!
+//! ```ignore
+//! [build-dependencies]
+//! substrate-wasm-builder = { ... }
+//! ```
+//!
+//! >Note that a runtime must always be one-runtime-per-crate. You cannot define multiple runtimes
+//! per rust crate.
+//!
+//! You can find the full code of this guide in [`first_runtime`].
+//!
+//! ## Your First Runtime
+//!
+//! The first new property of a real runtime that it must define its
+//! [`frame::runtime::prelude::RuntimeVersion`]:
+#![doc = docify::embed!("./packages/guides/first-runtime/src/lib.rs", VERSION)]
+//!
+//! The version contains a number of very important fields, such as `spec_version` and `spec_name`
+//! that play an important role in identifying your runtime and its version. More about runtime
```

Comment:
nit: This does not flow well, maybe sth like this?
```suggestion
//! that play an important role in identifying your runtime and its version, more importantly in runtime upgrades. More about runtime
```